src/core_modules/capture-core/components/Pages/Search/SearchForm/SearchForm.container.js

```diff
@@ -88,18 +89,19 @@ const mapDispatchToProps = (dispatch: ReduxDispatch, { searchGroupsForSelectedSc
     searchViaAttributesOnScopeProgram: ({ programId, formId, resultsPageSize }) => {
         dispatch(searchViaAttributesOnScopeProgram({ programId, formId, pageSize: resultsPageSize }));
     },
-    saveCurrentFormData: (searchScopeType, searchScopeId, formId, formsValues) => {
-        const currentSearchTerms =
-          collectCurrentSearchTerms(searchGroupsForSelectedScope, formsValues);
+    saveCurrentFormData: ({ searchScopeType, searchScopeId, formId, formsValues }) => {
+        const currentSearchTerms = collectCurrentSearchTerms(searchGroupsForSelectedScope, formsValues);
 
-        dispatch(actionCreator(searchPageActionTypes.CURRENT_SEARCH_INFO_SAVE)(
-            { searchScopeType,
-                searchScopeId,
-                formId,
-                currentSearchTerms,
-            }));
+        dispatch(saveCurrentSearchInfo({
+            searchScopeType,
+            searchScopeId,
+            formId,
+            currentSearchTerms,
+        }));
+    },
```

Comment:
Looks like I have forgotten to extract this. I am creating an action creator from this now